services/movie_session.py

```diff
@@ -0,0 +1,48 @@
+from db.models import MovieSession
+from django.utils.dateparse import parse_date
+from datetime import datetime
+
+
+def create_movie_session(
+        movie_show_time: str | datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> None:
+    return MovieSession.objects.create(

```

Comment:
Consider validating `movie_show_time` to ensure it is a valid `datetime` object. If it's a string, it should be parsed into a `datetime` object before being used.